notebooks/responsible_ai/explainable_ai/solutions/xai_text_with_LIT.ipynb

```diff
@@ -0,0 +1,2770 @@
+{
```

Comment:
<p>In the markdown, add a description of what LIT is "The Learning Interpretability Tool (LIT) is a&nbsp;<strong>visual, interactive ML model-understanding</strong>&nbsp;tool that supports text, image, and tabular data." and maybe provide a link to the page.</p><p> </p><p>nit: Change Lit -&gt; LIT so it is consistent across the notebook.</p><p> </p><p>For the implementation of the subclass "Dataset" can you add a description of what should go in the class?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>It's already written on the top. Why you want to repeat it?</p><p> </p><p>Changed every occurence to <code>LIT</code> !</p><p> </p><p>I added an additional line to explain this specific dataset object case (converting tf.data.Dataset -&gt; LIT dataset). Other requirements are already written I think. Let me know what you think!</p>